admin-auth/src/main/kotlin/com/epam/drill/admin/auth/route/UserAuthenticationRoutes.kt

```diff
@@ -42,6 +46,20 @@ object UpdatePassword
 @Location("/api/login")
 object Login
 
+fun StatusPages.Configuration.authStatusPages() {
+    exception<UserNotAuthenticatedException> { cause ->
+        logger.trace(cause) { "401 User is not authenticated" }
+        call.respond(HttpStatusCode.Unauthorized, "User is not authenticated")
+    }
+    exception<IncorrectCredentialsException> { cause ->
+        logger.trace(cause) { "401 Username or password is incorrect" }
+        call.respond(HttpStatusCode.Unauthorized, "Username or password is incorrect")
+    }
+    exception<UserValidationException> { cause ->
+        logger.trace(cause) { "400 User data is invalid" }
+        call.respond(HttpStatusCode.BadRequest, "User data is invalid")
+    }
+}
```

Comment:
can we add exception<Throwable> as a last clause that will handle or other exception types and'll set HTTP 500 code?

Reply:
add message + data into Exception classes and pass to response in exception<MyException> handling

Reply:
will be fixed in the next branch